playground/css-lightningcss/__tests__/css-lightningcss.spec.ts

```diff
@@ -61,7 +61,7 @@ test('css modules', async () => {
 
 test('inline css modules', async () => {
   const css = await page.textContent('.modules-inline')
-  expect(css).toMatch(/\.\w{6}_apply-color-inline/)
+  expect(css).toMatch(/\._?\w{6}_apply-color-inline/)
```

Comment:
lightningcss adds `_` if the hash started with digits.
https://github.com/parcel-bundler/lightningcss/blob/master/src/css_modules.rs#L543-L544